src/code/gDPLoadTextureBlock_Runtime.c

```diff
@@ -0,0 +1,63 @@
+#ifndef GDP_LOADTEXTUREBLOCK_RUNTIME_C
+#define GDP_LOADTEXTUREBLOCK_RUNTIME_C
```

Comment:
```suggestion
#ifndef GDP_LOADTEXTUREBLOCK_RUNTIME_INC_C
#define GDP_LOADTEXTUREBLOCK_RUNTIME_INC_C
```
Along with renaming the file